account_tax_repartition_line_tax_group_account/static/description/index.html

```diff
@@ -0,0 +1,447 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
```suggestion

```
XML declarations in HTML module descriptions are deprecated in Odoo 17

Reply:
It's adding with pre-commit config, need to change it there